framework/doc/content/sqa/framework_stp.md

```diff
@@ -1 +1,241 @@
 !template load file=stp.md.template category=framework project=Framework
+
+!template! item key=test-scope
+The scope of this plan is to outline the necessary steps for tagging a repository
+release suitable for deployment and long term support. While much of the testing
+for the [!ac](MOOSE) and MOOSE-based applications is automated, [!ac](NQA-1) guidelines
+require reviews and approvals for each official supported release suitable for
+deployment. This plan describes how automated testing fulfills many of the
+quality testing requirements of the software and how the developers and asset
+owners can leverage this information when certifying each release.
+
+[!ac](MOOSE) and MOOSE-based software is built into executables that can linked
```

Comment:
that can "be" linked (only cause I was reading it for my own info and noticed)

Reply:
Thanks - I'll be proofing it or having somebody else doing that soon!